keyboards/yampad/readme.md

```diff
@@ -0,0 +1,2 @@
+YAMPAD
+===
```

Comment:
```suggestion
# YAMPAD
```

This file should be filled out according to the [template](https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/readme.md).